# Размер скан-строки
uint16 size

# Скан-строка
uint8[] data_line